selectors.go

```diff
@@ -46,14 +52,14 @@ func parseYml(ymlFile []byte) map[uint64]uint64 {
 func EvmChainIdToChainSelector() map[uint64]uint64 {
 	copyMap := make(map[uint64]uint64, len(evmChainIdToChainSelector))
 	for k, v := range evmChainIdToChainSelector {
-		copyMap[k] = v
+		copyMap[k] = v.ChainSelector
 	}
 	return copyMap
 }
 
 func ChainIdFromSelector(chainSelectorId uint64) (uint64, error) {
 	for k, v := range evmChainIdToChainSelector {
```

Comment:
I think lib users should use accessing functions like `ChainIdFromSelector` instead of accessing the map directly. However, if they for some reason need that map then it's rebuilt every time requested because this is a defensive way to prevent altering the internal state. Returning a single precomputed map may lead to a problem in which if users want to add something to this map, they update it for all. A map is passed by a pointer/reference, not a value.